package.json

```diff
@@ -50,7 +50,7 @@
     "cids": "^0.6.0"
   },
   "dependencies": {
-    "@dashevo/dapi-client": "~0.8.0",
+    "@dashevo/dapi-client": "~0.9.0-dev.4",
```

Comment:
Maybe `dev.5`? I believe `dev.4` does not have latest `dapi-grpc`.

Reply:
Done 👍